dashboard/test/ui/features/flappydrag.feature

```diff
@@ -1,18 +1,10 @@
-Feature: Flappy blocks can be dragged 
-
-Background:
-  Given I am on "http://studio.code.org/flappy/1?noautoplay=true"
+Feature: Flappy blocks can be dragged
 
 Scenario: Connect two blocks from toolbox
+  Given I am on "http://studio.code.org/flappy/1?noautoplay=true"
   When I rotate to landscape
   And I wait for the page to fully load
-  And I drag block "1" to block "3"
-  And I drag block "1" to block "4"
-  And I wait for 1 seconds
-  Then block "5" is child of block "4"
-
-Scenario: Connect two blocks from toolbox
-  When I wait for the page to fully load
+  And I wait to see ".blocklySvg"
```

Comment:
this seems great -- nothing better than waiting for the element you definitely need to perform the next step.